okapi-core/src/main/java/org/folio/okapi/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,61 @@
+package org.folio.okapi.logging;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.impl.ContextInternal;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
+ */
+@Plugin(name = "FolioLoggingContext", category = StrLookup.CATEGORY)
+public class FolioLoggingContext implements StrLookup {
+
+  public static final String EMPTY_VALUE = "";
```

Comment:
If we have traditional (non-JSON) log lines we may need EMPTY_VALUE as placeholder.
For JSON log lines this is not needed. The key should be completely omitted: https://softwareengineering.stackexchange.com/questions/343710/use-empty-string-null-or-remove-empty-property-in-api-request-response

Reply:
There is no easy way to omit empty or null values.
Returning "" or adding a default value like `property.requestId =` to the log4j2.properties have the same result.